canal/sync.go

```diff
@@ -50,11 +50,7 @@ func (c *Canal) runSyncBinlog() error {
 
 		// Update the delay between the Canal and the Master before the handler hooks are called
 		c.updateReplicationDelay(ev)
-		// if log pos equal zero ,it is a fake rotate event,ignore it.
-		// see https://github.com/mysql/mysql-server/blob/8cc757da3d87bf4a1f07dcfb2d3c96fed3806870/sql/rpl_binlog_sender.cc#L899
-		if ev.Header.LogPos == 0 {
```

Comment:
why comment these out?

Reply:
friendly ping @mefcorvi

Reply:
Sorry, it was an attempt to work around the issue in #406; it's unrelated to this PR so I've reverted these changes.